src/app/public/plugin-resources/code-examples/text-expand/inline/text-expand-demo.component.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'app-text-expand-demo',
+  templateUrl: './text-expand-demo.component.html'
+})
+export class TextExpandDemoComponent {
+
+  // tslint:disable-next-line
+  public longText = 'The text expand component truncates long blocks of text with an ellipsis and a link to expand the full text. When users click the link, the component expands to display the full text inline unless it exceeds limits on text characters or newline characters. If the text exceeds those limits, then the link expands the full text in a modal view instead. The component does not truncate text that is shorter than a specified threshold, but it always truncates text that includes newline characters and removes those newline characters from the truncated text.';
```

Comment:
Same question here about the difference between the updated summary text and the demo text. Seems like they should be the same, but we should confirm that the changes to the summary are accurate. See the other comment for that summary text.

Reply:
Updated to match the other.